src/pages/contact.js

```diff
@@ -82,7 +82,7 @@ class ContactUs extends Component {
     const site = this.props.data.site
     const page = this.props.data.allContentfulPage.edges[0].node
     return (
-      <Layout>
+      <Layout location={this.props.location}>
```

Comment:
Maybe destructure this from props in the lines above, before passing it down?

Reply:
Do you want me to push up this little thing?

Then I reckon merge?

Reply:
Sure, I’m not in computer atm, if you can do it, go for it :)